src/services/meetings/meetings-util.ts

```diff
@@ -0,0 +1,86 @@
+import { SupabaseDB } from "../../supabase"; // Supabase database
+import {
+    meetingView,
+    createMeetingValidator,
+    updateMeetingValidator,
+} from "./meetings-schema";
+
+export const getAllMeetings = async () => {
+    console.log("Getting all meetings from Supabase");
+    const { data, error } = await SupabaseDB.MEETINGS.select("*");
+    if (error) throw error;
+
+    return data.map((m) =>
+        meetingView.parse({
+            meetingId: m.meeting_id,
+            committeeType: m.committee_type,
+            startTime: m.start_time,
+        })
+    );
+};
+
+export const getMeetingById = async (meetingId: string) => {
+    const { data, error } = await SupabaseDB.MEETINGS.select()
```

Comment:
same as above. select only what is needed. also be consistent with format, generally better to use function syntax imo for clarity